msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -102,11 +102,9 @@ For any other external deep linking scenarios, you can use `app.openLink`, which
 
 Starting with version 2.0 of TeamsJS, the Teams platform concept of [task module](../../task-modules-and-cards/what-are-task-modules.md) has been renamed to *dialog* for better consistency with existing concepts across the Microsoft 365 developer ecosystem. Accordingly, the `tasks` namespace has been deprecated in favor of the new `dialog` namespace.
 
-This new dialog capability is split into a main capability (`dialog`) for supporting HTML-based dialogs and a subcapability, `dialog.bot`, for bot-based dialog development.
+The new `dialog` capability is split into two subcapabilities: `dialog.url` for HTML-based dialogs, and `dialog.adaptiveCard` for Adaptive Card-based dialogs. Functions used for communication between an app and its dialog have been moved into the `dialog.url` subcapability (as these only ever worked for HTML-based dialogs). For Adaptive Card dialogs, a new function `microsoftTeams.getAdaptiveCardSchemaVersion` has been added to check what schema version is supported by the host currently running your app. If the host does not support Adaptive Cards, it will return `undefined`.
```

Comment:
```suggestion
The new `dialog` capability is split into two subcapabilities: `dialog.url` for HTML-based dialogs, and `dialog.adaptiveCard` for Adaptive Card-based dialogs. Functions used for communication between an app and its dialog have been moved into the `dialog.url` subcapability (as these only ever worked for HTML-based dialogs). For Adaptive Card dialogs, a new function `microsoftTeams.getAdaptiveCardSchemaVersion` has been added to check what schema version is supported by the host currently running your app. If the host doesn't support Adaptive Cards, it will return `undefined`.
```

Reply:
Global: Change does not to doesn't.